### PR TITLE
Toronto calendar patch

### PR DESCRIPTION
**Pull request by katieannemills**
This patch addresses some usage confusion with the new calendars feature, and removes a bug that incorrectly limits the number of event listings on the webpage; thanks to @minisciencegirl for catching this bug.

(also: @mbonsma your suggestion of a note about setting calendars to public has been included upstream - thanks!).

### CODE REVIEW

File: scripts/updateCalendar.py [hunk 1]
```diff
@@ -37,12 +37,10 @@
 except ImportError:
     flags = None
 
-
-
 # Modify these variables in step 2 above ------------------- 
 APPLICATION_NAME = 'test' # app name you created in step one above
 CALENDAR_ID='USER@gmail.com' # google account name you created for your calendar
-POSTS_DIRECTORY="/path/to/your/studyGroup/_posts" # full path to your studyGroup/_posts directory.
+POSTS_DIRECTORY="/path/to/your/studyGroup/_posts" # full path to your studyGroup/_posts directory on your computer (not on github).
```
Comment (QuLogic) [inline scripts/updateCalendar.py]: Wait, why can't this just be relative?

File: scripts/updateCalendar.py [hunk 2]
```diff
@@ -52,7 +50,6 @@
 DEFAULT_END_TIME = "16:30" # will be overridden by endTime in _posts
 REQUIRED_FIELDS = [ 'title', 'location', 'text', 'link', 'date' ]
 
-
```
Comment (QuLogic) [inline scripts/updateCalendar.py:55]: Two blank lines here should be PEP8 compliant (though I don't know if the rest of the file is or isn't.)

**Comment by katieannemills** [comment]
good call on a relative path for `_posts`; also made this _mostly_ pep8, up to some line lengths I disagree with.

File: scripts/updateCalendar.py [hunk 3]
```diff
@@ -37,27 +39,30 @@
...
-APPLICATION_NAME = 'test' # app name you created in step one above
-CALENDAR_ID='USER@gmail.com' # google account name you created for your calendar
-POSTS_DIRECTORY="/path/to/your/studyGroup/_posts" # full path to your studyGroup/_posts directory.
-TIME_ZONE_STR = 'America/Vancouver' # not sure? check here: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
-#-----------------------------------------------------------
+# Modify these variables in step 2 above -------------------
+# APPLICATION_NAME: app name you created in step one above:
+APPLICATION_NAME = 'test'
+# CALENDAR_ID: google account name you created for your calendar:
+CALENDAR_ID = 'mills.wj@gmail.com'
```
Comment (QuLogic) [inline scripts/updateCalendar.py]: Not sure if you want to leave this one as `USER` (obviously, we'll be changing it in any case.)

**Comment by katieannemills** [comment]
ah, silly - good catch again, thanks!

**Comment by QuLogic** [comment]
@mbonsma You should probably check this since you're administrating the calendar account.

**Pull request merged**